docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
+
+services:
+  dalai:
+    build: .
+    user: "1000:1000"
+    environment:
+      - LLAMA_MODEL=13B
+    volumes:
+      - .:/home/dalai/app
+      - ./models:/home/dalai/models
+    ports:
+      - 3000:3000
+    # command: tail -f /dev/null
```

Comment:
Should we add here again `command: yarn just:run $LLAMA_MODEL`?
When I run `docker compose up` the env variable in docker-compose.yaml is ignored and the hardcoded one from Dockerfile is used